addons/website/static/tests/interactions/snippets/chart.test.js

```diff
@@ -18,27 +15,28 @@ patch(Chart.prototype, {
     setup() {
         super.setup();
         this.noAnimation = true;
-    }
+    },
 });
 
 test("chart is started when there is an element .s_chart", async () => {
-    const { core } = await startInteractions(`
```

Comment:
Note: good practice for another PR: `startInteractions` should be made markup-aware, rather than assuming its parameter is safe.